tests/ApiControllerTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Cronbeat\Tests;
+
+use Cronbeat\Controllers\ApiController;
+use PHPUnit\Framework\Assert;
+
+class ApiControllerTest extends DatabaseTestCase {
+    private function call(ApiController $controller, string $uri): string {
+        $_SERVER['REQUEST_URI'] = $uri;
+        return $controller->doRouting();
+    }
+
+    public function testPingStartAndCompleteEndpoints(): void {
+        // Given
+        $db = $this->getDatabase();
+        $db->createUser('u', 'p');
+        $userId = $db->validateUser('u', 'p');
+        if ($userId === false) { Assert::fail('user validate failed'); }
+        $uuid = $db->createMonitor('m1', $userId);
+        if ($uuid === false) { Assert::fail('monitor create failed'); }
+        $controller = new ApiController($db);
```

Comment:
@jetbrains-junie no asserts in given, review other new tests added in this branch as well